autogpt_platform/backend/backend/server/v2/store/db.py

```diff
@@ -619,7 +657,7 @@ async def get_user_profile(
 
 async def update_profile(
     user_id: str, profile: backend.server.v2.store.model.Profile
-) -> backend.server.v2.store.model.CreatorDetails:
+) -> backend.server.v2.store.model.ProfileDetails:
```

Comment:
can't really figure out the diff between these two and why we'd return creator over profile here?

Reply:
idk i'm not changin it but it should be `ProfileDetails` I think